bugbug/similarity.py

```diff
@@ -155,7 +155,7 @@ def get_similar_bugs(self, query, k=10):
 
 class NeighborsSimilarity(BaseSimilarity):
     def __init__(self, k=10, vectorizer=TfidfVectorizer):
-        self.vectorizer = TfidfVectorizer()
+        self.vectorizer = vectorizer(ngram_range=(1, 2))
```

Comment:
I would make this an option. The caller can use whatever vectorizer they want, with a default of TfidfVectorizer.
The vectorizer can be a parameter of the __init__ function.
Either:
```
    def __init__(self, k=10, vectorizer_class=TfidfVectorizer):
        self.vectorizer = vectorizer_class()
```
or:
```
    def __init__(self, k=10, vectorizer=TfidfVectorizer()):
        self.vectorizer = vectorizer
```

Reply:
Isn't the first option you defined above same as what I'm doing right now?

Reply:
You are forcing the usage of ngram_range=(1, 2)